TenureInformationApi/V1/Controllers/TenureInformationController.cs

```diff
@@ -62,7 +62,9 @@ public async Task<IActionResult> GetByID([FromRoute] TenureQueryRequest query)
             var result = await _getByIdUseCase.Execute(query).ConfigureAwait(false);
             if (result == null) return NotFound(query.Id);
 
-            HttpContext.Response.Headers.Add(HeaderConstants.ETag, EntityTagHeaderValue.Parse($"\"{result.VersionNumber.Value}\"").Tag);
+            int versionNumber = (result.VersionNumber != null) ? result.VersionNumber.Value : 0;
```

Comment:
if the versionNumber is `null` from dynamo, it won't be `0`

Reply:
I would suggest this logic, which works:
```

var eTag = string.Empty;
if (person.VersionNumber.HasValue)
    eTag = person.VersionNumber.ToString();

HttpContext.Response.Headers.Add(HeaderConstants.ETag, EntityTagHeaderValue.Parse($"\"{eTag}\"").Tag);
```

Reply:
So it should return "" if there is no version?

Reply:
yep.